lumen/ai/assistant.py

```diff
@@ -155,9 +164,15 @@ def download_notebook():
             notebook_button, *self.sidebar_widgets, self._current_agent, Tabs(("Memory", memory))
         )
 
-    def _add_suggestions_to_footer(self, suggestions: list[str], inplace: bool = True):
+    def _add_suggestions_to_footer(
```

Comment:
This is now tweaked to show applicable custom analyses to the current pipeline as well.